pep-9999.rst

```diff
@@ -514,6 +569,53 @@ No::
         forward_reference: 'OtherClass'
     class OtherClass: ...
 
+Version and Platform Checks
+---------------------------
+
+Use the tuple-based syntax for version checks. This is easier to read and write
+than the alternatives. Version checks should only check against the ``major``
+and ``minor`` parts of ``sys.version_info``.
+
+Yes::
+
+    if sys.version_info >= (3,):
+        # Python 3-specific type hints.
+    else:
+        # Python 2-specific type hints.
+
+    if sys.version_info >= (3, 5):
+        # Specific minor version features can be easily checked with tuples.
+
+    if sys.version_info < (3,):
+        # This is only necessary when a feature has no Python 3 equivalent.
+        # "<= (2, 7)" does not work because e.g. (2, 7, 1) > (2, 7).
+
+No::
+
+    if sys.version_info >= (3, 5, 1):
+        # Version checks should only check major and minor version numbers.
+
+    if sys.version_info[0] >= 3 and sys.version_info[1] >= 5:
```

Comment:
These two examples may not be needed anymore, if we're not supporting the syntax.

Reply:
The two sections serve slightly different purposes. Supported Constructs says type checkers don't need to support .major. The Style Guide says type stub authors should use tuple checks and to avoid per-element/.major checks. I think having both is important.

Reply:
That's fair.